developer-extension/src/common/types.ts

```diff
@@ -15,4 +17,12 @@ export interface Store {
   useRumSlim: boolean
   logEventsFromRequests: boolean
   blockIntakeRequests: boolean
+  local: {
+    [tabId: number]: LocalStore
```

Comment:
Is the localStore cleaned up at some point?

Reply:
not that I know of.
FMU, tab ids are likely to be reused so it should not grow indefinitely.

any concerns?